scalahost/nsc/src/test/scala/scala/meta/tests/scalahost/mirrors/SemanticSuite.scala

```diff
@@ -235,4 +235,14 @@ class SemanticSuite extends DatabaseSuite {
       |[294..294) [C[Int]]
   """.trim.stripMargin
   )
+  messages(
+    """
+      |import scala.collection.mutable.{ Map, Set, ListBuffer }
+      |object a { ListBuffer.empty[Int] }
+    """.stripMargin.trim,
+    """
+      |[0..34..56): [warning] Unused import
```

Comment:
cc/ @gabro this is how Position.RangeWithPoint looks like, the middle number is the `.point` that is where the caret `^` appears in scalac error messages.